isis/src/control/objs/ControlNet/ControlNet.h

```diff
@@ -221,6 +237,25 @@ namespace Isis {
       friend class ControlPoint;
 
     public:
+      struct Image {
+        QString serial;
+        QList<ControlMeasure* > measures;
```

Comment:
When we remove measures from the network, we'll need to be find them in these lists and then remove them. Because this list can get quite large, we probably want to use something that is faster to remove things from such as a a map or hash.